esmvalcore/config-developer.yml

```diff
@@ -201,7 +201,7 @@ ACCESS:
     default:
       - '{dataset}/{sub_dataset}/{exp}/{modeling_realm}/netCDF'
   input_file:
-    default: '{sub_dataset}.{special_attr}-*.nc'
+    default: '{sub_dataset}.{special_attr}-{regular_expression}*.nc'
```

Comment:
Again in `/g/data/p73/archive/non-CMIP/ACCESS-ESM1-5/ESM-CDR-01/history/ocn`
I have `ocean_month.nc.XXXX`
That does not match the pattern above...